jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -376,6 +379,16 @@ class DeploymentFlow extends React.Component {
     });
   }
 
+  /**
+    Check whether this should be an expert flow.
+
+    @method _isExpertFlow
```

Comment:
don't need to define the method anymore :)

Reply:
Removed.